tests/test_evpn_multicast.rb

```diff
@@ -0,0 +1,56 @@
+# Evpn Multicast Unit Tests
+#
+# Rahul Shenoy, January, 2018
+#
+# Copyright (c) 2018 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/evpn_multicast'
+require_relative '../lib/cisco_node_utils/feature'
+
+# TestEvpnMulticast - Minitest for EvpnMulticast class
+class TestEvpnMulticast < CiscoTestCase
+  @skip_unless_supported = 'evpn_multicast'
+
+  def setup
```

Comment:
We have seen issues in subsequent tests if we don't include a `teardown` method.   Suggest you add one to remove `feature ngmvpn` and `advertise evpn multicast` when you are done.